vendor/assets/javascripts/brainstem/brainstem-model.coffee

```diff
@@ -95,6 +105,16 @@ class window.Brainstem.Model extends Backbone.Model
     else
       super(field)
 
+  set: (key, val, options) ->
+    super
+
+    _.extend(@unsavedAttributes || {}, @changed)
```

Comment:
No, it kind of breaks `@unsavedAttributes` in the case of instantiating the model with attributes (kind of edge case because usually you don't pass `id` in which case all attributes would be persisted).